src/plugins/chat.ts

```
@@ -48,6 +54,805 @@ interface StreamedResponseContentChunk {
+            },
+            sticker: {
+                type: 'string',
+                description: 'HTTP(S) sticker URL'
+            },
+            image: {
+                type: 'string',
+                description: 'HTTP(S) image URL'
+            },
+            parts: {
+                type: 'array',
+                description: 'Multiple parts inside one message, joined in order.',
```

ESLint

[error] Insert `⏎···················`

(prettier/prettier)